build/dockerfiles/rhel.Dockerfile

```diff
@@ -50,16 +50,8 @@ RUN ./check_mandatory_fields.sh devfiles
 
 RUN ./index.sh > /build/devfiles/index.json
 RUN ./list_referenced_images.sh devfiles > /build/devfiles/external_images.txt
-RUN chmod -R g+rwX /build/devfiles
-
-
-FROM registry.access.redhat.com/ubi8/nodejs-16-minimal:1-6 as dwtemplates
-USER 0
```

Comment:
the next 4 lines are already done in the builder stage, so by using node16-mini image, as builder, we don't need a second stage here with 4 duplicate steps.